src/components/Favorites/index.js

```diff
@@ -1,20 +1,59 @@
-import { useState, useEffect } from "react";
+import { useEffect } from "react";
+import {
+    importSavedFavorites,
+    removeAllExistingFavorites,
+    removeExistingFavorite,
+} from "../../features/favoritesSlice";
 import styles from "./favorites.module.css";
+import { useDispatch, useSelector } from "react-redux";
+import { AiOutlineStar } from "react-icons/ai";
 
 function Favorites() {
+    const { data } = useSelector((state) => state.favorites);
+    const dispatch = useDispatch();
     useEffect(() => {
-        // on page load...
-        // check if we have any favorites stored in local storage
-        // if yes, pull them -> set them in redux favorites
-        // if no, do nothing
-    }, []);
+        if (!data?.length) {
+            dispatch(importSavedFavorites());
+        }
+    }, [dispatch, data]);
+    const clearFavorites = () => {
+        dispatch(removeAllExistingFavorites());
+        dispatch(importSavedFavorites());
+    };
+    const removeFavorite = () => {
+        dispatch(removeExistingFavorite());
+        dispatch(importSavedFavorites());
+    };
 
     // if favorites, render them out
-    // if (favorites.data?.length) {
-    // return the favorites
-    // }
-    // if not, leave empty
-    return <div />;
+    if (data?.length) {
+        // return the favorites
+        // }
+        // if not, leave empty
+        console.log("favorites", data);
+        return (
+            <section className={styles.sidePanel}>
+                <h2>Favorites</h2>
+                <ul>
+                    {data.map((coin, index) => (
+                        <li key={index}>
+                            {coin}
```

Comment:
Bad practice to just render text without a wrapping element. use a `span` for inline non paragraph text blocks like this. It will need a `key` assigned to it as well.